model/project_parser_merge_functions.go

```diff
@@ -180,36 +180,6 @@ func (pp *ParserProject) mergeUnique(toMerge *ParserProject) error {
 		pp.ExecTimeoutSecs = toMerge.ExecTimeoutSecs
```

Comment:
Sure that new naming makes sense

Reply:
I think as long as we keep it so that there is no way to include file paths on the project config struct, that guarantees that the configs have to exist in the top level yaml.  Would it be necessary / helpful to allow for merging of the ProjectConfig struct? The number of properties it supports is relatively small.

Reply:
I think we'll just need to be really clear in the documentation, because users don't have a concept of "ParserProject struct" and "ProjectConfig struct", they just have a yaml file. Probably it would be okay to enforce that all of these things be defined at the top level for now and this would simplify things for us, but we'll just need to be clear.